alignak/daemons/arbiterdaemon.py

```diff
@@ -546,16 +546,48 @@ def manage_missing_daemons(self):
                 ret = self.my_satellites[daemon_name].poll()
                 if ret is not None:
                     logger.error("*** %s exited on start!", daemon_name)
-                    for line in iter(self.my_satellites[daemon_name].stdout.readline, b''):
-                        logger.error(">>> " + line.rstrip())
-                    for line in iter(self.my_satellites[daemon_name].stderr.readline, b''):
-                        logger.error(">>> " + line.rstrip())
+                    if self.my_satellites[daemon_name].stdout:
+                        for line in iter(self.my_satellites[daemon_name].stdout.readline, b''):
+                            logger.error(">>> " + line.rstrip())
+                    if self.my_satellites[daemon_name].stderr:
+                        for line in iter(self.my_satellites[daemon_name].stderr.readline, b''):
+                            logger.error(">>> " + line.rstrip())
                     result = False
                 else:
                     logger.info("%s running (pid=%d)",
                                 daemon_name, self.my_satellites[daemon_name].pid)
         return result
 
+    def stop_missing_daemons(self):
```

Comment:
This function has many problems:
* in comment, it's written `stop the daemon` but in code it just look if stopped, not try stop it
* in comment we have `True if all daemons are stopped, else False`, but in code it return True only in case all daemons are running